models/article.models.js

```diff
@@ -5,10 +5,19 @@ exports.fetchTopics = () => {
     return rows;
   });
 };
+// author: expect.any(String),
+// title: expect.any(String),
+// article_id: expect.any(Number),
+// topic: expect.any(String),
+// created_at: expect.any(String),
+// votes: expect.any(Number),
 
 exports.fetchArticleById = async (id) => {
   return db
-    .query(`SELECT * FROM articles WHERE article_id = $1;`, [id])
+    .query(
+      `SELECT articles.author, title, articles.article_id, articles.body, articles.topic, articles.created_at, articles.votes, COUNT(comment_id)::INT AS comment_count FROM articles JOIN comments ON comments.article_id=articles.article_id WHERE articles.article_id = $1 GROUP BY comments.article_id, articles.author, title, articles.article_id, articles.body, articles.topic, articles.created_at, articles.votes;`,
+      [id]
+    )
```

Comment:
Very nice! Bear in mind that with a regular JOIN - we'll be abandoning any articles that DO NOT have any associated comments